src/unlimited.cpp

```diff
@@ -693,12 +693,42 @@ int isChainExcessive(const CBlockIndex* blk, unsigned int goBack)
     return (recentExcessive && !oldExcessive);
 }
 
-bool CheckExcessive(const CBlock& block, uint64_t blockSize, uint64_t nSigOps, uint64_t nTx)
+bool CheckExcessive(const CBlock& block, uint64_t blockSize, uint64_t nSigOps, uint64_t nTx, uint64_t largestTx)
 {
-    if (blockSize > excessiveBlockSize) {
+    if (blockSize > excessiveBlockSize) 
+    {
         LogPrintf("Excessive block: ver:%x time:%d size: %" PRIu64 " Tx:%" PRIu64 " Sig:%d  :too many bytes\n", block.nVersion, block.nTime, blockSize, nTx, nSigOps);
         return true;
     }
+ 
+    if (blockSize > BLOCKSTREAM_CORE_MAX_BLOCK_SIZE)
+      {
+	// Check transaction size to limit sighash
+        if (largestTx > maxTxSize.value)
+          {
```

Comment:
Has anyone tried the contrib/devtools/clang-format.py script?
We have all those nice settings in src/.clang-format.